src/reporter.js

```diff
@@ -102,8 +102,13 @@ function CoverageIstanbulReporter(baseReporterDecorator, logger, config) {
       .map;
 
     if (!coverageConfig.skipFilesWithNoCoverage) {
+      // On Windows, istanbul returns files with mixed forward/backslashes in them
+      const fixedFilePaths = {};
+      remappedCoverageMap.files().forEach(path => {
+        fixedFilePaths[util.fixPathSeparators(path)] = true;
+      });
       coverageMap.files().forEach(path => {
-        if (!(path in remappedCoverageMap)) {
+        if (!(util.fixPathSeparators(path) in fixedFilePaths)) {
```

Comment:
You have to module.exports fixPathSeparators in util.js file i think

Reply:
Ugh... I missed checking that file in... oops!